WS2012R2/lisa/setupscripts/STOR_VHDXResize_Basic.ps1

```diff
@@ -154,12 +160,13 @@ Write-Output "Covers: ${TC_COVERED}" | Tee-Object -Append -file $summaryLog
 # Convert the new size
 #
 $newVhdxSize = ConvertStringToUInt64 $newSize
-
+$sizeFlag = ConvertStringToUInt64 "50GB"
```

Comment:
what is the role of the new var sizeFlag? The 'newvhdxsize' variable is currently used and parsed from the xml